ts/components/IdpSuccessfulAuthentication.tsx

```diff
@@ -2,28 +2,54 @@
  * A component to display a white tick on a blue background
  */
 import * as React from "react";
-import { View, StatusBar, StyleSheet } from "react-native";
+import { View, StyleSheet, SafeAreaView } from "react-native";
 
-import { IOIconSizeScale, Icon } from "@pagopa/io-app-design-system";
-import variables from "../theme/variables";
+import {
+  ContentWrapper,
+  H3,
+  IOStyles,
+  Pictogram,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { profileNameSelector } from "../store/reducers/profile";
+import I18n from "../i18n";
+import { useOnFirstRender } from "../utils/hooks/useOnFirstRender";
+import { trackIdpAuthenticationSuccessScreen } from "../screens/profile/analytics";
+import { useIOSelector } from "../store/hooks";
+import { loggedInIdpSelector } from "../store/reducers/authentication";
 
 const styles = StyleSheet.create({
   container: {
-    backgroundColor: variables.brandPrimary,
-    flex: 1,
-    alignItems: "center",
-    justifyContent: "center"
+    ...IOStyles.centerJustified,
+    ...IOStyles.flex
   }
 });
 
-const ICON_SIZE: IOIconSizeScale = 96;
-
-export const IdpSuccessfulAuthentication = () => (
-  <View style={styles.container}>
-    <StatusBar
-      barStyle="light-content"
-      backgroundColor={styles.container.backgroundColor}
-    />
-    <Icon name="checkTickBig" color="white" size={ICON_SIZE} />
-  </View>
-);
+export const IdpSuccessfulAuthentication = () => {
+  const idp = useIOSelector(loggedInIdpSelector);
+  useOnFirstRender(() => {
+    trackIdpAuthenticationSuccessScreen(idp?.id);
+  });
+  const name = useIOSelector(profileNameSelector);
+  const contentTitle = I18n.t("authentication.idp_login_success.contentTitle", {
+    name
+  });
+  return (
+    <SafeAreaView style={styles.container}>
+      <ContentWrapper>
+        <View style={{ alignItems: "center" }}>
+          <Pictogram name="success" size={120} />
+          <VSpacer size={24} />
```

Comment:
We want to use a constant to store this value?

Reply:
As above.